test/repositories.test.js

```diff
@@ -13,7 +13,17 @@ describe('repository test', () => {
 
   it('Get user by email', async () => {
     const userRepo = makeUserRepository()
-    const user = await userRepo.getUserByEmail('user@gmail.com')
+    const user = await userRepo.getUserBy('email', 'user@gmail.com')
+    expect(user).to.have.all.keys('id', 'email', 'password', 'nickname')
+  })
+  it('Create user', async () => {
+    const userRepo = makeUserRepository()
+    const user = await userRepo.createUser({ email: 'new@gmail.com', password: 'newpass', nickname: 'newnickname' })
+    expect(user).to.have.all.keys('id', 'email', 'password', 'nickname')
+  })
+  it('Update user', async () => {
+    const userRepo = makeUserRepository()
+    const user = await userRepo.updateUser({ param: 'password', value: 'newpassword', email: 'user2@gmail.com' })
```

Comment:
What if in the future I'll want to change password by nickname?
I think would be better if `updateUser` will take two params:
first object to search:
`
{
   <field serach>: value 
}
`
second object to set:
`
{
  <field to set>: <value> 
}
`
for example: 
`updateUser({nickname: "test"}, {email: "new.email@gamail.com"}})`
here I want to change email in user with nickname `test`  to `new.email@gmail.com`